### ISSUE TITLE
"Remove braces from single expression" false positive for "unsafe" block

### ISSUE THREAD

**Issue by andy-hanson** [intellij-rust/intellij-rust#1797]
```rust
let slice = &[1];
let start = slice.as_ptr();
let end = unsafe { start.offset(1) };
```
Put your cursor on the `unsafe` keyword -- there is a suggestion to "Remove braces from single expression". But I apply the quickfix, I will get a compile error `error[E0133]: call to unsafe function requires unsafe function or block`.

**Comment by vlad20012** [comment]
The actual code is located [here](https://github.com/intellij-rust/intellij-rust/blob/fcb26cbdbd7ae7a654597413b52862f3e115f58b/src/main/kotlin/org/rust/ide/intentions/UnwrapSingleExprIntention.kt#L21) (and the tests [here](https://github.com/intellij-rust/intellij-rust/blob/fcb26cbdbd7ae7a654597413b52862f3e115f58b/src/test/kotlin/org/rust/ide/intentions/UnwrapSingleExprIntentionTest.kt))

We should just add one addition check `if (blockExpr.unsafe != null) return null`

**Comment by maraisr** [comment]
Can I grab this one @vlad20012
